### PR TITLE
Ensure all requires are tested

### PR DESCRIPTION
**Pull request by oraclown**

### CODE REVIEW

File: test/testParachainStaking.t.sol [hunk 1]
```diff
@@ -69,6 +69,18 @@ contract ParachainStakingTest is Test {
...
         assertEq(address(staking.registryAddress()), address(registry));
         assertEq(address(staking.governance()), address(0x2));
+
+        // Try to create new w/o passing in token address
+        vm.prank(bob);
+        vm.expectRevert("must set token address");
+        ParachainStaking _ps = new ParachainStaking(address(registry), address(0x0));
+    }
+
+    function testInit() public {
```
Comment (tkernell) [inline test/testParachainStaking.t.sol:79]: should also test other 2 requires:

- call init again after contract already init'ed
- input governance address as address 0

File: test/testParachainStaking.t.sol [hunk 2]
```diff
@@ -201,21 +346,47 @@ contract ParachainStakingTest is Test {
...
-        // Wait for lock period to expire (7 days after staker start date)
-        vm.warp(block.timestamp + 7 days + 1 seconds);
         // Withdraw stake
         staking.withdrawParachainStake(fakeParaId);
         assertEq(token.balanceOf(address(staking)), 0);
         assertEq(token.balanceOf(address(paraOwner)), 100);
 
         vm.stopPrank();
+
+        // todo: how to test transfer fail?
```
Comment (tkernell) [inline test/testParachainStaking.t.sol:356]: This should in theory never fail if our contract is built correctly. You could however use a custom token contract which allows you to edit a given address' balance, and then lower the contract's bal below the transfer amount.

File: test/testParachainGovernance.t.sol [hunk 3]
```diff
@@ -85,6 +85,22 @@ contract ParachainGovernanceTest is Test {
         assertEq(address(gov.token()), address(token));
```
Comment (tkernell) [inline test/testParachainGovernance.t.sol:85]: should test registry address, teamMultisig address, owner address

File: test/testParachainGovernance.t.sol [hunk 4]
```diff
@@ -85,6 +85,22 @@ contract ParachainGovernanceTest is Test {
...
     }
 
+    function testInit() public {
```
Comment (tkernell) [inline test/testParachainGovernance.t.sol:88]: should test token and staking addresses

**Review by tkernell — APPROVED** [review]

File: test/testParachainStaking.t.sol (hunk 2 above)
Comment (oraclown) [reply test/testParachainStaking.t.sol:356]: Removed

File: test/testParachainGovernance.t.sol (hunk 4 above)
Comment (oraclown) [reply test/testParachainGovernance.t.sol:88]: Done

File: test/testParachainGovernance.t.sol (hunk 3 above)
Comment (oraclown) [reply test/testParachainGovernance.t.sol:85]: owner already tested. Added check for registry and multisig addresses

**Pull request merged**